src/controller_examples/rabbitmq_controller/exec/reconciler.rs

```diff
@@ -732,11 +749,8 @@ fn make_service_account(rabbitmq: &RabbitmqCluster) -> (service_account: Service
             }
             owner_references
         });
-        metadata.set_labels({
-            let mut labels = StringMap::empty();
-            labels.insert(new_strlit("app").to_string(), rabbitmq.name().unwrap());
-            labels
-        });
+        metadata.set_labels(make_labels(rabbitmq));
```

Comment:
@euclidgame It seems to me you never update the service account, role and role binding. In that case you might not set the labels and annotations to them since the two fields are not immutable. Otherwise you will have difficulty in writing liveness property for them (their labels/annotations won't be consistent with your cr)

Reply:
We should update those objects later.